tools/tracer/gui/Program.cs

```diff
@@ -1,14 +1,22 @@
-﻿/* ****************************************************************************** *\
-
-INTEL CORPORATION PROPRIETARY INFORMATION
-This software is supplied under the terms of a license agreement or nondisclosure
-agreement with Intel Corporation and may not be copied or disclosed except in
-accordance with the terms of that agreement
-Copyright(c) 2012-2020 Intel Corporation. All Rights Reserved.
-
-File Name: Program.cs
-
-\* ****************************************************************************** */
+// Copyright (c) 2015-2022 Intel Corporation
```

Comment:
Should data be aligned with previous one, like 2015-2022?

Reply:
done